cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -67,7 +80,12 @@ def _cleanup():
         migration_plan.name, view.migration_plans_completed_list.get_vm_count_in_plan(
             migration_plan.name), view.migration_plans_completed_list.get_clock(
             migration_plan.name))
+    # validate MAC address matches between source and target VMs
     assert view.migration_plans_completed_list.is_plan_succeeded(migration_plan.name)
+    src_vm = form_data_vm_obj_single_datastore.vm_obj[0]
```

Comment:
You already have this named as `src_vm_obj`, why re-name it and index the fixture list again?